Configuration/Eras/python/Era_Run3_ppRef_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_cff import Run3
+from Configuration.Eras.Modifier_ppRef_run3_cff import ppRef_run3
+from Configuration.Eras.ModifierChain_trackingMkFitProd_cff import trackingMkFitProd
+from Configuration.ProcessModifiers.trackingNoLoopers_cff import trackingNoLoopers
+
+Run3_ppRef = cms.ModifierChain(Run3.copyAndExclude([trackingMkFitProd, trackingNoLoopers]), ppRef_run3)
```

Comment:
I'm not sure why you're trying to add a dedicated pp ref era.
We don't use any such thing in the reconstruction.
pp ref uses standard pp ref settings.
We don't need to introduce any dependence on Configuration/Eras

Reply:
I did this following old examples. I can remove the era and do all the configuration elsewhere.